solver/src/main.rs

```diff
@@ -191,6 +192,13 @@ struct Arguments {
     /// and not to be included in the objective function by the HTTP solver.
     #[structopt(long, env, use_delimiter = true)]
     liquidity_order_owners: Vec<H160>,
+
+    /// Fee scaling factor for objective value. This controls the constant
+    /// factor by which order fees are multiplied with. Setting this to a value
+    /// greater than 1.0 makes settlements with negative objective values less
+    /// likely, promoting more aggressive merging of single order settlements.
+    #[structopt(long, env, default_value = "1", parse(try_from_str = shared::arguments::parse_fee_factor))]
+    pub fee_objective_scaling_factor: f64,
 }
```

Comment:
I was under the impression this constant term used to make negative objective positive was going to be an additive constant. However, I guess this works too...